public/dota2.html

```diff
@@ -49,7 +49,10 @@ <h1>DOTA 2</h1>
             }
           }
         ));
-        var superResolution = (window.superResolution = examplePlayer.superResolution({"weights": dota2_weights}));
+        let superResolution = (window.superResolution = examplePlayer.superResolution({
+          "weights": dota2_weights,
+          "frameRate": 30,
```

Comment:
This one is natively 60, right? I think it's probably fine at 30fps, but we might want to think about whether 1080p@30 is better than, say, 720p@60. It probably depends on the kind of content.

Reply:
Oops, thanks!